wiki/People/Beatmap_Nominators/en.md

```diff
@@ -72,73 +72,90 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
 
 | Name                                                  | Additional Languages     |
 |-------------------------------------------------------|--------------------------|
+| [- Matha -](https://osu.ppy.sh/users/7354729) | Chinese, Cantonese |
+| [\_Epreus](https://osu.ppy.sh/users/7342798) | Korean |
+| [Agatsu](https://osu.ppy.sh/users/5579871) |  |
 | [ailv](https://osu.ppy.sh/users/6933054)              |                          |
 | [Andrea](https://osu.ppy.sh/users/33599)              | Italian                  |
+| [Arf](https://osu.ppy.sh/users/3716999) |  |
 | [Beomsan](https://osu.ppy.sh/users/3626063)           | Korean                   |
+| [Bibbity Bill](https://osu.ppy.sh/users/4446810) |  |
 | [bossandy](https://osu.ppy.sh/users/360437)           | Chinese                  |
-| [DeRandom Otaku](https://osu.ppy.sh/users/5156153)    | Urdu                     |
+| [celerih](https://osu.ppy.sh/users/4696296) | French |
+| [Cris-](https://osu.ppy.sh/users/6175280) | Spanish |
+| [Crissa](https://osu.ppy.sh/users/5405836) | Spanish |
 | [Doormat](https://osu.ppy.sh/users/3230571)           |                          |
 | [DTM9 Nowa](https://osu.ppy.sh/users/5428909)         | Finnish                  |
+| [eiri-](https://osu.ppy.sh/users/3388410) | Spanish |
 | [Electoz](https://osu.ppy.sh/users/6485263)           | Thai                     |
+| [Error-](https://osu.ppy.sh/users/3378391) | Indonesian |
 | [fieryrage](https://osu.ppy.sh/users/3533958)         |                          |
-| [Garden](https://osu.ppy.sh/users/2849992)            | Chinese                  |
 | [Gero](https://osu.ppy.sh/users/1467715)              | Spanish                  |
 | [Hailie](https://osu.ppy.sh/users/5226970)            |                          |
+| [Hanasaka Yui](https://osu.ppy.sh/users/6443117) |  |
+| [Hinsvar](https://osu.ppy.sh/users/1249323) | Indonesian |
 | [hypercyte](https://osu.ppy.sh/users/9155377)         |                          |
 | [IceKalt](https://osu.ppy.sh/users/5410645)           | German                   |
-| [iYiyo](https://osu.ppy.sh/users/3919785)             | Spanish, some German     |
 | [jonathanlfj](https://osu.ppy.sh/users/270377)        | Chinese, some French     |
 | [Kaitjuh](https://osu.ppy.sh/users/2225327)           | Dutch                    |
 | [Kalibe](https://osu.ppy.sh/users/3376777)            | Polish                   |
+| [Krfawy](https://osu.ppy.sh/users/1478292) |  |
```

Comment:
Additional language for Krfawy: Polish

Reply:
done on all 4 languages to all your suggestions